test/support/spec.rb

```diff
@@ -1434,3 +1436,60 @@ def run_specs
 end
 
 at_exit { run_specs }
+
+
+# NATFIXME method to wrap spec-matchers that are known to fail and mark them
+# as "fix this later".
+#
+# Can be used in the following ways:
+#
+# 1) hide any StandardError inside of the block
+# NATFIXME "some description" do
+#   some_spec.should be "bad"
+# end
+#
+# 2) Hide any error matching a klass
+# NATFIXME "some description", klass: NoMethodError do
+#   some_spec.foo.should be "also bad"
+# end
+#
+# 3) Hide any error matching a klass and with error-message match
+# NATFIXME "some description", klass: NoMethodError, match: /method foo undefined/ do
+#   some_spec.foo.should be "also bad"
+# end
+#
+# If the code inside the block does not cause the error or the error it caues does not match the klass (or klass/match) then
+# NatalieFixMeException will be raised.
+#
+def NATFIXME(description, klass: nil, match: nil)
```

Comment:
I think we should rename the `match` argument to `message`.

Reply:
And maybe rename `klass` to `exception` as well? I think class is a bit too broad of a term.

Reply:
Updated both kwargs in the latest push